AuctionTrading/Infrastructure/AuctionTrading.Infrastructure.EntityFramework/Configurations/SellerConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿using AuctionTrading.Domain.Entities;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore;
+using AuctionTrading.Domain.ValueObjects;
+
+namespace AuctionTrading.Infrastructure.EntityFramework.Configurations
+{
+    public class SellerConfiguration : IEntityTypeConfiguration<Seller>
+    {
+        public void Configure(EntityTypeBuilder<Seller> builder)
```

Comment:
@oleggolen 
При первой миграции выдает предупреждение
The foreign key property 'AuctionLot.SellerId1' was created in shadow state because a conflicting property with the simple name 'SellerId' exists in the entity type, but is either not mapped, is already used for another relationship, or is incompatible with the associated primary key type. See https://aka.ms/efcore-relationships for information on mapping relationships in EF Core.
Что может быть не так?

Reply:
@Yana-Kryukova 
А вы пробовали заменить: builder.HasMany("_auctionLots").WithOne();
на: builder.HasMany<AuctionLot>("_auctionLots").WithOne(e => e.Seller);
Как я писал в телеграмме во вторник?
Или это не помогло?

Reply:
В здешних комментариях, текст внутри угловых скобках автоматически скрывается, если специально не заморочиться, но в данном случае важны именно угловые скобки и их содержимое, поэтому подправил, чтобы отображалось корректно

Reply:
@m-a-ilnitsky к сожалению не помогает...

Reply:
А сообщение об ошибке меняется?
И если просто удалить builder.HasMany("_auctionLots").WithOne(); из SellerConfiguration что-то меняется?

Reply:
@Yana-Kryukova 
В общем я не понял в чем здесь проблема... но почему-то какие-то ваши имена не нравятся EF-у.
У меня в ровно той же ситуации такой проблемы нет.
Возможно просто в каком-то имени есть опечатка, например, русская буква вместо английской, но я её не смог найти, может Олег увидит это в Rider-е, там это проще обнаружить.
Или сочетание слов EF-у не нравится.
В общем причина проблемы не понятна.

Но решение я нашёл.

Нужно в SellerConfiguration написать так:
builder.HasMany<AuctionLot>("_auctionLots")
    .WithOne(e => e.Seller)
    .HasForeignKey("SellerId")
    .HasPrincipalKey(e => e.Id);

Reply:
@m-a-ilnitsky  СПАСИБО!!! Все супер, все сработало. Я тоже думала про явное указание внешнего ключа, но не знала про то, что в угловых скобках можно указывать конкретный тип связанного объекта... поэтому не получалось

Reply:
@Yana-Kryukova Ну у меня тоже самое нормально сработало и без явного указания ключей, только с указанием типа в угловых скобках, как было в первом предложенном варианте...
И кстати, в последнем варианте сайт опять скрыл угловые скобки, а я не проверил и только сейчас заметил.
Но я так понимаю, что вы догадались, что там должны были быть фигурные скобки